index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio-setup-and-mobile-first</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="menu">
+        <a href="#">My Logo</a>
+        <div class="menu-bar">
+          <div id="menu-bar-1"></div>
+          <div id="menu-bar-2"></div>
+          <div id="menu-bar-3"></div>
+        </div>
+      </nav>
+      <section class="main-section">
+        <div class="content">
```

Comment:
- Great job for using semantic tags, 👍👍 I think It is better to use a section outside the `header`. kindly use the below format 👇
```
<header>
</header>
<main>
   <section>
        .......
   </section>
</main>
<footer>
</footer>
````